PrimeRust/solution_1/src/main.rs

```diff
@@ -332,7 +325,7 @@ pub mod primes {
     ) {
         if show_results {
             eprint!("2,");
-            for num in (3..prime_sieve.sieve_size).filter(|n| prime_sieve.is_num_flagged(*n)) {
+            for num in (3..prime_sieve.sieve_size).filter(|n| prime_sieve.flags.get(*n)) {
```

Comment:
This needs the old `is_num_flagged` function, because
- `is_num_flagged` takes the number as input
- `flags.get` takes an index in the storage array (not a number)

Reply:
You are right, I was too hasty again.
Now, should I use the `is_num_flagged` function or would it be okay to translate between number and flag index, as seen in the prime number search?

Reply:
I think we can keep the old function there, and just use it for the reporting.